packages/contracts/contracts/Unirep.sol

```diff
@@ -585,6 +591,21 @@ contract Unirep is IUnirep, VerifySignature {
         return attester.epochLength;
     }
 
+    function attesterEpochSealed(uint160 attesterId, uint256 epoch)
```

Comment:
we can use it in
https://github.com/Unirep/Unirep/blob/4b12237a1fb8bec43ca6ff57a224b17617238cec/packages/contracts/contracts/Unirep.sol#L338-L342